PROCESS.md

```diff
@@ -0,0 +1,279 @@
+they can be fast-tracked through public code review later at disclosure
+time.
+
+### Draft advisory
+
+In the mean time, the HSRT coordinator prepares a security advisory
+that will be communicated to downstream stakeholders.
+
+The description should properly credit the reporter, specify affected
+versions (including unsupported ones) and accurately describe impact and
+mitigation mechanisms. The HSRT coordinator should use the template
+below.
+
+### Review advisory
+
+The advisory is validated by the reporter and the project's maintainer.
+
+### Send CVE request
+
+To ensure full traceability, we attempt to obtain a CVE assignment
+before the issue is communicated to a larger public. This is generally
+done as the patch gets nearer to final approval. The approved advisory
+is submitted through [MITRE's CVE Request form](https://cveform.mitre.org/).
+The *request type* is `Request a CVE ID`, the *e-mail address* should be
+that of the requester (generally the assigned HSRT coordinator),
+and for embargoed reports the coordinator's OpenPGP key should be pasted
+into the field provided.
+
+In the *required* section set the checkboxes indicating the product is
+not CNA-covered and that no prior CVE ID has been assigned, select an
+appropriate *vulnerability type* (using `Other or Unknown` to enter a
+freeform type if there is nothing relevant on the drop-down), set the
+*vendor* to `Haskell`, and the *product* and *version* fields to match
+the `$PROJECTS` and `$AFFECTED_VERSIONS` from the advisory. In
+the *optional* section set the radio button for *confirmed/acknowledged*
+to `Yes`, choose an appropriate *attack type* in the drop-down (often
+this is `Context-dependent` for our cases), check the relevant *impact*
+checkboxes, attempt to fill in the *affected components* and *attack
+vector* fields if possible, paste in the *suggested description* from
+the prose of the advisory (usually omitting the first sentence
+as it's redundant with other fields), put the `$CREDIT` details in the
+*discoverer/credits* field, and the bug URL in the *references* field.
+If the report is still private, note that in the *additional information*
+field like
+`This report is currently under embargo and no disclosure date has been scheduled at this time.`
+
+At the bottom of the page, fill in the *security code* and click the
+*submit request* button. If some fields contain invalid data they will
+be highlighted red; correct these, update the *security code* and
+*submit request* again until you get a confirmation page.
+
+### Get assigned CVE
+
+MITRE returns the assigned CVE. It is added to the advisory,
+and the bug is retitled to `$TITLE ($CVE)`.
+
+### Embargoed disclosure
+
+Once the patches are approved and the CVE is assigned, a signed email
+with the vulnerability description is sent to the downstream
+stakeholders. The disclosure date is set to 3-5 business days, excluding
+Monday/Friday and holiday periods, at 1500 UTC. No stakeholder is
```

Comment:
This seems too specific for us, at this stage.  Maybe we can just say that the notice will state the planned disclosure date.